vasyl_varyshchuk/Week_2/task_1/func_in_threads.py

```diff
@@ -0,0 +1,84 @@
+""""
+This module returns random size of file name from list by processing elements in separate threads
+"""
+
+import random
+import string
+from threading import Thread
+
+
+number_of_threads = 3
+length_of_list = number_of_threads * 100
+
+
+def recognize_file_type(file_name):
+    """This function returns extension of file passing file name as parameter"""
+    _, extension = file_name.split('.')
+    return extension
+
+
+def generate_file_names_list():
+    """
+    This function generate list of files with random names.
+    Length of list is specified by global variable
+    len_of_list which value depend on number of thread.
+    """
+    extensions = ['txt', 'zip', 'ini', 'jpg', 'ttf']
+    file_names = []
+    for i in range(length_of_list):
+        file_name = ''.join([random.choice(string.ascii_lowercase) for j in range(5)])
+        full_file_name = '.'.join([file_name, random.choice(extensions)])
```

Comment:
It's not clear from the name what is `full_file_name`.

Reply:
Ok, I changed this name in last commit. I think now it looks more clearly.